utils/doclint/generateDotnetApi.js

```diff
@@ -407,7 +407,7 @@ function generateNameDefault(member, name, t, parent) {
           attemptedName = `${parent.name}BoundingBoxResult`;
         if (attemptedName === 'BrowserContextCookie')
           attemptedName = 'BrowserContextCookiesResult';
-        if (attemptedName === 'File')
+        if (attemptedName === 'File' || attemptedName === 'SetValue')
```

Comment:
SetValue may become non-unique in the future, consider checking parent name too.

Reply:
SG